tapdb/asset_minting.go

```diff
@@ -543,6 +606,70 @@ func fetchAssetSeedlings(ctx context.Context, q PendingAssetStore,
 			EnableEmission: dbSeedling.EmissionEnabled,
 		}
 
+		if dbSeedling.TweakedScriptKey != nil {
+			tweakedScriptKey, err := btcec.ParsePubKey(
+				dbSeedling.TweakedScriptKey,
+			)
+			if err != nil {
+				return nil, err
+			}
+
+			scriptKeyInternalPub, err := btcec.ParsePubKey(
+				dbSeedling.ScriptKeyRaw,
+			)
+			if err != nil {
+				return nil, err
+			}
+
+			scriptKeyLocator := keychain.KeyLocator{
+				Index: extractSqlInt32[uint32](
+					dbSeedling.ScriptKeyIndex,
+				),
+				Family: extractSqlInt32[keychain.KeyFamily](
+					dbSeedling.ScriptKeyFam,
+				),
+			}
+
+			seedling.ScriptKey = &asset.ScriptKey{
+				PubKey: tweakedScriptKey,
+				TweakedScriptKey: &asset.TweakedScriptKey{
+					RawKey: keychain.KeyDescriptor{
+						KeyLocator: scriptKeyLocator,
+						PubKey:     scriptKeyInternalPub,
+					},
+					Tweak: dbSeedling.ScriptKeyTweak,
+				},
+			}
+		}
+
+		if dbSeedling.GroupKeyRaw != nil {
```

Comment:
We should update the unit tests assertions to make sure these are always set as expected.